sky/tools/create_macos_binary.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
licenses check wants this added:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8747664324136339489/+/u/test:_licenses_check/stdout

and pylint has opinions:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8747664264773789985/+/u/test:_pylint/stdout

Reply:
Hitting up all the classics. I should have left room for a `gn format` to get the trifecta.